HIP/hip-0000-expand-alias-crypto-create-transfer.md

```diff
@@ -0,0 +1,225 @@
+
+As an illustration, a wallet or exchange could create and fund a new Hedera account based on the `public-address` alone. Once the transaction goes through the owner of the `public-address` user may use a wallet of their choice to submit a signed transaction (e.g. `CryptoTransfer`, `ContractCreate`, `ContractCall`, `EthereumTransaction` or `TokenAssociation`) to claim ownership of the account and carry out the desired transaction. The ownership claim and account completion will be seamless and transparent to the user.
+
+`CryptoTransfer` should also support this case. This will support developers and exchanges who prefer to transfer to and create an account in one step.
+
+### Allow CryptoCreate to automatically set alias
+
+When `CryptoCreate.key` is an `ECDSA` key, the network node may calculate the `public-address` and automatically set the accounts alias value. 
+
+## Backwards Compatibility
+
+Existing accounts are unaffected by the expansion and the additional alias field on the transaction protobuf is already supported.
+
+`CryptoTransfer` auto-account creation is un-impacted and will still support the provision of the public key. 
+
+### Mirror Node Impications
+
+The Mirror Node can already ingest account creation transactions with alias values.
+This will only increate the applicable parent trnasactions that can cause this flow.
+Where it was limited to CryptoTransfer it will now include CryptoCreate
+
+### Wallet/Exchange Implications
+
+With the suggested changes, the alias property of an account will become more applicable and will need greater visibility.
+
+To provide greater web3 user scenario support Wallets & Exchanges should
+
+- Display the account alias property of an account
+- Create an ECDSA based account with alias when full smart contract support is desired
+- Accept the use of an alias as a account identifier in HBAR and token transfers
+- Accept the use of an alias during account creation
+- Add support to check if an account exists with a given alias
+
+### Performance Implications
+
+The mass adoption of alias setting on new and existing accounts will have minor but notable memory implications on the Hedera ecosystem
+
+`CryptoCreate` transactions will see extra bytes for every account created with an alias
+
+The size of the bytes in each case depends on the governing account key type. For an `ECDSA` based account the alias will always be a valid Ethereum address and will be 20 bytes long. In the `ED25519` case the alias will be a valid protobuf serialized public key of 35 or 68 bytes depending on encoding.
```

Comment:
Are we going to support ED25519 key aliases for CryptoCreate? See related discussion: https://github.com/hashgraph/hedera-services/pull/3943#discussion_r972471310.